I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 183, in script call
    call episode3
  File "game/episode 3.rpy", line 163, in script
    n "+ - * % ** {space=40}Arithmetic Operators"
TypeError: * wants int

-- Full Traceback ------------------------------------------------------------

Traceback (most recent call last):
  File "game/script.rpy", line 183, in script call
    call episode3
  File "game/episode 3.rpy", line 163, in script
    n "+ - * % ** {space=40}Arithmetic Operators"
  File "renpy/ast.py", line 2915, in execute
    Say.execute(self)
    ~~~~~~~~~~~^^^^^^
  File "renpy/ast.py", line 991, in execute
    renpy.exports.say(who, what, *args, **kwargs)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/exports/sayexports.py", line 118, in say
    what = what % tag_quoting_dict
           ~~~~~^~~~~~~~~~~~~~~~~~
TypeError: * wants int

Windows-11-10.0.22621-SP0 AMD64
Ren'Py 8.4.1.25072401
Codebound 1.0
Thu Oct 30 16:28:13 2025
